src/apps/search-header/search-header.dev.tsx

```diff
@@ -9,11 +9,16 @@ export default {
   title: "Apps / Search Header",
   component: SearchHeaderEntry,
   argTypes: {
-    altText: {
+    searchHeaderIconAltText: {
       name: "Alt text for search button image",
       defaultValue: "search icon",
       control: { type: "text" }
     },
+    searchHeaderInputLabel: {
+      name: "Search header input label",
+      defaultValue: "The main search field used to search for literature.",
```

Comment:
Consider removing the specific reference to literature. The municipal libraries provide many other types of materials 😅 .

Reply:
Very fair :) I haven't thought of that.